agent/lib/kontena/service_pods/creator.rb

```diff
@@ -153,7 +153,7 @@ def volumes_exist?(service_pod)
 
       # @param [Docker::Container] container
       def cleanup_container(container)
-        container.stop('timeout' => 10)
+        container.stop('timeout' => container.stop_grace_period)
```

Comment:
IMO no. One might be deploying newer version of an app that handles `SIGTERM` differently than the existing container and then stopping it with different timeout might create some issues.